Build Error

I got these Errors as below when `catkin build als_ros`.


```bash
Errors     << als_ros:make /home/gisen/ros_workspaces/alk_ws/logs/als_ros/build.make.002.log
CMakeFiles/classifier_dataset_generator.dir/src/classifier_dataset_generator.cpp.o: 関数 `als_ros::ClassifierDatasetGenerator::ClassifierDatasetGenerator()' 内:
classifier_dataset_generator.cpp:(.text._ZN7als_ros26ClassifierDatasetGeneratorC2Ev[_ZN7als_ros26ClassifierDatasetGeneratorC5Ev]+0x1a4): `cv::Mat::Mat()' に対する定義されていない参照です
CMakeFiles/classifier_dataset_generator.dir/src/classifier_dataset_generator.cpp.o: 関数 `als_ros::ClassifierDatasetGenerator::mapCB(boost::shared_ptr<nav_msgs::OccupancyGrid_<std::allocator<void> > const> const&)' 内:
classifier_dataset_generator.cpp:(.text._ZN7als_ros26ClassifierDatasetGenerator5mapCBERKN5boost10shared_ptrIKN8nav_msgs14OccupancyGrid_ISaIvEEEEE[_ZN7als_ros26ClassifierDatasetGenerator5mapCBERKN5boost10shared_ptrIKN8nav_msgs14OccupancyGrid_ISaIvEEEEE]+0x22d): `cv::Mat::Mat(int, int, int)' に対する定義されていない参照です
classifier_dataset_generator.cpp:(.text._ZN7als_ros26ClassifierDatasetGenerator5mapCBERKN5boost10shared_ptrIKN8nav_msgs14OccupancyGrid_ISaIvEEEEE[_ZN7als_ros26ClassifierDatasetGenerator5mapCBERKN5boost10shared_ptrIKN8nav_msgs14OccupancyGrid_ISaIvEEEEE]+0x4ed): `cv::Mat::Mat(int, int, int)' に対する定義されていない参照です
collect2: error: ld returned 1 exit status
make[2]: *** [/home/gisen/ros_workspaces/alk_ws/devel/.private/als_ros/lib/als_ros/classifier_dataset_generator] Error 1
make[1]: *** [CMakeFiles/classifier_dataset_generator.dir/all] Error 2
make[1]: *** 未完了のジョブを待っています....
CMakeFiles/mae_classifier_learning.dir/src/mae_classifier_learning.cpp.o: 関数 `als_ros::ClassifierDatasetGenerator::ClassifierDatasetGenerator()' 内:
mae_classifier_learning.cpp:(.text._ZN7als_ros26ClassifierDatasetGeneratorC2Ev[_ZN7als_ros26ClassifierDatasetGeneratorC5Ev]+0x1a4): `cv::Mat::Mat()' に対する定義されていない参照です
collect2: error: ld returned 1 exit status
make[2]: *** [/home/gisen/ros_workspaces/alk_ws/devel/.private/als_ros/lib/als_ros/mae_classifier_learning] Error 1
make[1]: *** [CMakeFiles/mae_classifier_learning.dir/all] Error 2
make: *** [all] Error 2
```

I think ${OpenCV_LIBRARIES} should be add to target_link_libraries in CMakeLists.txt.
I would send PR, please confirm it.

--- a/src/als_ros/CMakeLists.txt
+++ b/src/als_ros/CMakeLists.txt
@@ -37,10 +37,10 @@ add_executable(gl_pose_sampler src/gl_pose_sampler.cpp)
 target_link_libraries(gl_pose_sampler ${catkin_LIBRARIES} ${OpenCV_LIBRARIES})
 
 add_executable(classifier_dataset_generator src/classifier_dataset_generator.cpp)
-target_link_libraries(classifier_dataset_generator ${catkin_LIBRARIES})
+target_link_libraries(classifier_dataset_generator ${OpenCV_LIBRARIES} ${catkin_LIBRARIES})
 
 add_executable(mae_classifier_learning src/mae_classifier_learning.cpp)
-target_link_libraries(mae_classifier_learning ${catkin_LIBRARIES} yaml-cpp)
+target_link_libraries(mae_classifier_learning ${OpenCV_LIBRARIES} ${catkin_LIBRARIES} yaml-cpp)
 
 add_executable(evaluator src/evaluator.cpp)
 target_link_libraries(evaluator ${catkin_LIBRARIES})

